src/lib/tabs.ts

```diff
@@ -0,0 +1,90 @@
+import * as Messaging from "@src/lib/messaging"
+
+const allTabs = -1
+
+// Small wrapper meant to enable sending a message either to a single tab or
+// multiple ones. Note that for now, sending messages to all tabs does not
+// work, for reasons unknown.
+const msg = (tabId, ...args) => {
+    if (tabId === allTabs) {
+        return Messaging.messageAllTabs("omniscient_content", ...args)
```

Comment:
`jsb tri.tabs[-1].tri.excmds.js("console.log('lol')")` works for me - what were you trying?

Reply:
I tried `:jsb tri.tabs.console.log("hello world")`, which did not work because of an obvious mistake. Thanks for the nudge, it's fixed now!